examples/cmd/examples.go

```diff
@@ -31,7 +31,7 @@ func init() {
 
 func newSDK() (*sdk.SDK, error) {
 	resolver.SetDefaultScheme("passthrough")
-	opts := []sdk.Option{sdk.WithStoreCollectionHeaders()}
+	opts := []sdk.Option{sdk.WithStoreCollectionHeaders(), sdk.WithInsecurePlaintextConn()}
```

Comment:
you probably want to drop sdk.WithStoreCollectionHeaders().

There is no way to determine if a nanoTDF is part of a collection or not, with this option on, it will always assume its part of a collection.

Since you are decrypting the same nanoTDF, it will store the header and skip the rewrap call.

It's probably best that you remove this option for benchmarking.

Reply:
I've added a parameter, `--storeCollectionHeaders`, to the `examples` command. It defaults to `true`. I've also updated the nano benchmark in CI has it set to `false`